src/pages/Hero/styles.ts

```diff
@@ -1,5 +1,31 @@
 import styled from 'styled-components'
+import { theme } from '../../styles'
 
 export const Content = styled.div`
+    
+`
 
+export const InfoCard = styled.div`
+    align-items: flex-start;
+    background-color: ${theme.background2};
```

Comment:
utilizar a prop do theme
```suggestion
    background-color: ${(props) => props.theme.background2};
```

Reply:
Done